ruby-gem/lib/calabash-android/operations.rb

```diff
@@ -644,7 +644,8 @@ def start_test_server_in_background(options={})
         if keyguard_enabled?
           wake_up
         end
-
+        app_tries = options[:app_launch_tries] || 100
+        options.delete(:app_launch_tries) unless options[:app_launch_tries].nil?
```

Comment:
It should be something like, change `env_options = options` to `env_options = options.dup`, and remove the : app_launch_tries key using `env_options.delete(:app_launch_tries)`, no need for an if.